src/rh-footer-links-wrapper.ts

```diff
@@ -36,90 +39,93 @@ export class RhFooterLinkWrapper extends LitElement {
 
       :host([is-mobile]) {
         --link-font-size: 16px;
+        --rh-footer--links-columns: 1;
+        --rh-footer-links-flex-direction: row;
+        --rh-footer-links-width: 100% !important;
+        --rh-footer-link-width: 100% !important;
       }
 
-      .base {
-        width: 100%;
+      /* Add a touch of padding to the bottom of pfe-accordion-panels */
+      :host([is-mobile]) ::slotted(rh-footer-links) {
+        padding-bottom: 16px;
       }
 
-      .link {
-        display: inline-flex;
+      .base {
         width: 100%;
-        gap: var(--pf-global--spacer--lg, 24px);
-        margin-bottom: var(--pf-global--spacer--md, 16px);
       }
 
       @media screen and (min-width: ${mobileBreakpoint}) {
-        .link {
-          width: calc(50%);
+        :host([is-mobile]) {
+          --rh-footer-link-width: calc(50% - 5px) !important;
         }
       }
     `;
   }
 
-  firstUpdated() {
-    this.build();
-  }
-
   async build(): Promise<void> {
     // get a list of rh-footer-links items
     if (this.shadowRoot) {
-      const children = this.shadowRoot
-        .querySelector('slot')
-        ?.assignedElements({ flatten: true });
-      if (children && children.length > 0) {
-        const linkSets: LinkSet[] | undefined = this.shadowRoot
-          .querySelector('slot')
-          ?.assignedElements({ flatten: true })
-          .map(item => ({
-            // for each header we need to create an array of panel items that it's associated with.
-            header: item.querySelector('[slot="header"]'),
-            // collect all of the headers siblings
-            panel: [...item.children]
-              .filter(child => child.getAttribute('slot') !== 'header')
-              .map(child => {
-                // ensure it has a class of .link
-                child.classList.add('link');
-                // ensure it has a part name of link
-                child.setAttribute('part', 'link');
-                return child;
-              }),
-          }));
+      // update the lightdom
+      if (this.isMobile) {
+        const linkSets: LinkSet[] | undefined = [
+          ...this.querySelectorAll('rh-footer-links'),
+        ].map(item => ({
+          // for each header we need to create an array of panel items that it's associated with.
+          header: item.querySelector('[slot="header"]'),
+          panel: item as HTMLElement,
+        }));
+        // store the linkSets to the local state
+        this.linkSets = linkSets;
 
-        if (linkSets) {
-          this.linkSets = linkSets;
+        // if mobile then we need to apply the dynamic slots
+        for (const [index, set] of Object.entries(linkSets)) {
```

Comment:
to reduce cognitive load, maybe these (80-104) could be moved to `update()` (i.e. an `@observed` `_linkSetsChanged`)